drivers/hmis/lib/form_data/assessments/base_assessment.json

```diff
@@ -1592,6 +1601,7 @@
         },
         {
           "type": "CHOICE",
+          "component": "RADIO_BUTTONS",
```

Comment:
Changed default behavior so `NoYesReasonsForMissingData` doesn't default to radio buttons. Need to specify them explicitly.

Radio buttons are defaulted if there are fewer than 4 options.

Reply:
We should probably fill in `COMPONENT` for everything, so we don't have to change these when the default changes